src/libclipper/include/clipper/containers.hpp

```diff
@@ -31,13 +32,14 @@ class ModelContainer {
 
   size_t get_batch_size(Deadline deadline);
   double get_average_throughput_per_millisecond();
-  void update_throughput(size_t batch_size, long total_latency);
+  void update_container_stats(size_t batch_size, long total_latency);
   void send_feedback(PredictTask task);
 
   VersionedModelId model_;
   int container_id_;
   int replica_id_;
   InputType input_type_;
+  clipper::metrics::Histogram latency_hist_;
```

Comment:
This histogram (which receives datapoints here: https://github.com/ucbrise/clipper/pull/225/files#diff-628067215a2c6ded52652fff0b43720bR46) is not used now, but I could see it being involved in some selection policies / task discarding in the future. Happy to remove if this isn't the right time to include it.